solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -451,6 +451,51 @@ contract LiquidityProtection is TokenHandler, Utils, Owned, ReentrancyGuard, Tim
         return addProtectedLiquidity(_owner, poolToken, _baseToken, poolTokenAmount / 2, _amount);
     }
 
+    /**
+     * @dev returns the single-side staking limits of a given pool
+     *
+     * @param _poolAnchor   anchor of the pool
+     * @return base maximum amount of base token liquidity that can be single-sided staked in the pool
+     * @return network maximum amount of base token liquidity that can be single-sided staked in the pool
+     */
+    function poolAvailableSpace(IConverterAnchor _poolAnchor) external view returns (uint256, uint256) {
+        // get the pool token
```

Comment:
I think that this method should have the `poolSupported(_poolAnchor)` and `poolWhitelisted(_poolAnchor)` checks, so that it won't cause unexpected results for other converters

Reply:
Done